src/ManagedUsers/ManagedUsersFilter.php

```diff
@@ -318,27 +317,27 @@ public function setIncludeCitizenProjectHosts(?bool $value): void
     }
 
     /**
-     * @return ReferentTag[]
+     * @return Zones[]
      */
-    public function getReferentTags(): array
+    public function getManagedZones(): array
     {
-        return $this->referentTags;
+        return $this->managedZones;
     }
 
-    public function addReferentTag(ReferentTag $referentTag): void
+    public function addManagedZone(Zone $zone): void
     {
-        $this->referentTags[] = $referentTag;
+        $this->managedZones[] = $zone;
     }
 
-    public function removeReferentTag(ReferentTag $referentTag): void
+    public function removeManagedZone($zone): void
```

Comment:
typehint missing

Reply:
addressed in aecc91a7d9c2e379f23e6daa4e4f4c9e99abfce6